lai_s3/s3.py

```diff
@@ -1,41 +1,48 @@
 import botocore.exceptions
 import lightning as L
-from lightning.storage.payload import Payload
+import lightning_app as lapp
```

Comment:
We shouldn't be using the lightning_app import. We should have core functionality with 
```
import lightning as L
L.LightningWork()
L.LightningApp()

# For thing like Payload that should come from
from lightning.app.storage.payload import Payload
```

Reply:
Why is it bad to use lapp?

Reply:
Consistency